src/index.ts

```diff
@@ -1,29 +1,19 @@
-// we need this file because of this issue: https://github.com/fastify/fastify-cli/issues/131
-import 'make-promises-safe'
 import Fastify from 'fastify'
-import hospitalRun from './app'
+import app from './app'
 
 const port = Number(process.env.PORT) || 3000
-const ip = process.env.IP || '0.0.0.0'
+const ip = process.env.IP || 'localhost'
 
-const fastify = Fastify(hospitalRun.options)
-fastify.register(hospitalRun)
+const fastify = Fastify({
+  ignoreTrailingSlash: true,
+  logger: true,
+})
 
-if (process.env.NODE_ENV !== 'production') {
-  // eslint-disable-next-line
-  const blipp = require('fastify-blipp')
-  fastify.register(blipp)
-}
+fastify.register(app)
 
-fastify.listen(port, ip, err => {
+fastify.listen(port, ip, (err) => {
   if (err) {
-    fastify.log.error(err)
```

Comment:
`fastify.log.error(err)` makes use of fastify to log it properly. We have to avoid using console.log since we need to support advance logging.